src/software/backend/robot_status.h

```diff
@@ -84,3 +86,12 @@ struct RobotStatus
     // The FPGA bitstream build ID.
     uint32_t fpga_build_id;
 };
+
+/**
+ * Converts RobotStatus to TbotsRobotMsg
+ * Does not convert ErrorCode, ChipperKickerStatus,
```

Comment:
You should be able to set error code from `std::string robot_messages`

Reply:
comment should be updated

Reply:
Done.